src/vs/workbench/contrib/debug/browser/debugService.ts

```diff
@@ -255,7 +255,7 @@ export class DebugService implements IDebugService {
 	 * main entry point
 	 * properly manages compounds, checks for errors and handles the initializing state.
 	 */
-	startDebugging(launch: ILaunch | undefined, configOrName?: IConfig | string, noDebug = false, parentSession?: IDebugSession): Promise<boolean> {
+	startDebugging(launch: ILaunch | undefined, configOrName?: IConfig | string, noDebug = false, parentSession?: IDebugSession, options?: IDebugSessionOptions): Promise<boolean> {
```

Comment:
Putting parentSession and options and even noDebug together here in an option bag would make the code cleaner.